### PR TITLE
chore: tuning resource usage for operator pod

### PR DESCRIPTION
**Pull request by zdtsw**
- reduce cpu and mem usage
- request is  some data from testing in a large cluster psi-04
- limit is data from attached file in jira 9806
- 

## Description

related to https://issues.redhat.com/browse/RHOAIENG-9806

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by adelton** [comment]
Is this related to https://issues.redhat.com/browse/RHOAIENG-9806, as some preliminary stop-gap?

> some data from testing in a large cluster psi-04

Wouldn't data from small clusters be more relevant for the actual value we aim for?

Do we need for example CPU `requests` at all? What functionality will suffer when the Operator becomes CPU-starved?

And vice versa -- do we need to lower the limit at all?

File: bundle/manifests/opendatahub-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1730,10 +1730,10 @@ spec:
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 500m
-                    memory: 4Gi
+                    cpu: 200m
+                    memory: 2Gi
                   requests:
-                    cpu: 500m
+                    cpu: 25m
```
Comment (VaishnaviHire) [inline bundle/manifests/opendatahub-operator.clusterserviceversion.yaml:1736]: I think we should only update requests instead of limits.

**Comment by VaishnaviHire** [comment]
I agree with @adelton to use data from small clusters to set defaults. The jira issue linked has data from PSAP team

**Comment by zdtsw** [comment]
tbh, when i started this PR, i did not know this jira ticket. 
Mainly was from some test we did for another case. Then I recalled we had an old issue regarding resource utilization enhancement, so I submitted this PR after we finalized certain tests.

one thing on my mind after reading your comments:
for ticket https://issues.redhat.com/browse/RHOAIENG-9806 , should we use the same data from pref test in ODH?
I would assume these data were collected from downstream build. we can use it to set for downstream but how you feel we should use the same value in ODH (if it is not for the sake of sync code)

File: bundle/manifests/opendatahub-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (zdtsw) [reply bundle/manifests/opendatahub-operator.clusterserviceversion.yaml:1736]: i am confused for this part. I've left some question/comments in the jira and slack, would be good to understand these data first.

**Comment by adelton** [comment]
> 
> one thing on my mind after reading your comments: for ticket https://issues.redhat.com/browse/RHOAIENG-9806 , should we use the same data from pref test in ODH? I would assume these data were collected from downstream build. we can use it to set for downstream but how you feel we should use the same value in ODH (if it is not for the sake of sync code)

For the benefit of the folks who might not have access to the internal information, it might be useful to get the data from an ODH installation and share them here or in some other public place, so that the reasons for the numerical changes are documented. I would assume the numbers from ODH and downstream don't differ much, so if we can use and publish the numbers we got for downstream really depends on whether they are considered internal-only or not.

**Comment by adelton** [comment]
> 
> I dont think so, but more for https://issues.redhat.com/browse/RHOAIENG-9806

And specifically https://issues.redhat.com/browse/RHOAIENG-10889, it seems.

**Review by adelton — APPROVED** [review]
LGTM.

**Pull request merged**